README.md

```diff
@@ -1,36 +1,71 @@
-
-### Initial Setup
-1. Run `docker-compose run --rm django ./manage.py migrate`
-2. Run `docker-compose run --rm django ./manage.py createsuperuser`
-   and follow the prompts to create your own user
-
-### Run Application
-1. Run `docker-compose up`
-2. Access the site, starting at http://localhost:8000/
-   * Outgoing emails are sent to the console 
-3. The Django admin interface is still available at http://localhost:8000/admin/
-3. When finished, use `Ctrl+C`
-
-### Testing
-1. Run `docker-compose run --rm django tox`
-1. Run `docker-compose run --rm tailwind yarn lint --fix`
-
-### Application Maintenance
-Occasionally, new package dependencies or schema changes will necessitate
-maintenance. To non-destructively update your development stack at any time:
-1. Run `docker-compose build --pull --no-cache`
-2. Run `docker-compose run --rm django ./manage.py migrate`
-
-#### Install Javascript packages
-In the event that Javascript packages need to be installed, this can be done via Docker Compose:
-1. Run: `docker-compose run --rm tailwind yarn add -D package-name`,
-   substituting `package-name` as appropriate.
-
-This can also be done natively by running Yarn commands from the `tailwind/` directory, but be sure
-to re-build Docker afterwards (via Application Maintenance above).
+django-girder-style is a Django library providing
+styled Django templates for data management applications.
+
+
+### Benefits
+django-girder-style provides an extensible block-oriented base HTML template.
+This base template includes
+a pre-built [Tailwind CSS](https://tailwindcss.com/) (with some minor customizations) stylesheet,
+[Remix Icon](https://remixicon.com/) support,
+and the [Nunito](https://fonts.google.com/specimen/Nunito) font.
+
+Additionally, django-girder-style provides styled versions of all
+[django-allauth](https://django-allauth.readthedocs.io/) view templates.
+This styling allows some limited branding customization as well.
+
+## Installation
+Install django-girder-style:
+```bash
+pip install django-girder-style
+```
+
+Enable django-girder-style as an installed Django app:
+```python
+# settings.py
+INSTALLED_APPS = [
+    # Any project-local apps should come before "girder_style",
+    # so templates can be overridden as needed
+    'my_django_app.apps.MyDjangoAppConfig',
+    ...,
+    'girder_style',
+    ...,
+    # If "allauth" is installed, it must come after "girder_style"
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
+]
+```
+
+## Usage
+### Base Template
+All project templates
+[should extend](https://docs.djangoproject.com/en/3.1/ref/templates/language/#template-inheritance)
+`base.html`.
+This provides the following blocks to inject content:
+* `head_title`: The content of the `<title>` tag.
+* `extra_head`: Additional HTML placed within the `<head>` tag.
+* `body`: The entire HTML body content, including the `<body>` tag itself.
+
+For example, a template `my_app/home.html` may contain:
+```html
```

Comment:
```suggestion
```django
```

Reply:
It will show as Jinja, but that's better: https://github.com/github/linguist/blob/736df2bb2a78ccc23115cdeb33aef31156f3efac/lib/linguist/languages.yml#L2681